### DISCUSSION TITLE
actions buttons in last tab

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2352] [Help]
is it possible to put the action buttons for create in a last tab?

**Comment by danharrin** [comment]
You want to make a wizard?

**Reply by jcgmalave** [reply]
exactly, thats the idea

**Reply by danharrin** [reply]
Wizard is the next feature that we will release

**Reply by danharrin** [reply]
Probably in the next couple of weeks

**Reply by R1NEJ0** [reply]
I can't wait to that feature!!!
